spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/hive/HivePropertiesConverter.java

```diff
@@ -83,7 +93,17 @@ public Map<String, String> toGravitinoTableProperties(Map<String, String> proper
 
   @Override
   public Map<String, String> toSparkTableProperties(Map<String, String> properties) {
-    return toOptionProperties(properties);
+    Map<String, String> sparkTableProperties = toOptionProperties(properties);
+    String externalValue =
```

Comment:
the default value of `HivePropertiesConstants.GRAVITINO_HIVE_TABLE_TYPE` is not `false`, and we shouldn't depend on it.  could you rewrite like this?
```java
String hiveTableType = sparkTableProperties.get(HivePropertiesConstants.GRAVITINO_HIVE_TABLE_TYPE)
if (HivePropertiesConstants.GRAVITINO_HIVE_EXTERNAL_TABLE.equalsIgnoreCase(hiveTableType)) {
  xxx
}
```

Reply:
Updated. Please help to review again, thanks!!!!!!!!